priv/templates/phx.gen.release/rel/migrate.sh.eex

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd -P -- "$(dirname -- "$0")"
+./<%= otp_app %> eval <%= app_namespace %>.Release.migrate
```

Comment:
I am not a bash especialist, but should we use `exec` here to avoid the additional process?

```suggestion
exec ./<%= otp_app %> eval <%= app_namespace %>.Release.migrate
```